tools/FileTestDefs.sh

```diff
@@ -666,6 +666,33 @@ check_throws_test() {
 }
 standard_checks+=(check_throws)
 
+# Check for typos similar to Doxygen groups
+check_dox_groups() {
+    is_c++ "$1" && staged_grep -E "^\s*//+\s*[\{\}\(\)]*@[\{\}\(\)]*" "$1" \
+            | grep -v -E "(^\s*///@\{|^\s*///@\})"
+}
+check_dox_groups_report() {
+    echo "Found a likely typo similar to a valid doxygen grouping"
+    echo "Use balanced ///@{ and ///@} (on their own lines) to"\
+         " create doxygen groups"
+    pretty_grep -E "^\s*//+\s*[\{\}\(\)]*@[\{\}\(\)]*" "$1" | \
+        grep -v -E "(///@{|///@})"
+}
+check_dox_groups_test() {
+    test_check pass foo.hpp '///@{'
+    test_check pass foo.hpp '///@}'
```

Comment:
(Why) do you prefer to omit the space after `///`? Is `/// @{` also valid?

Reply:
Yes, both are valid, but I was sufficiently irritated by the number of times we got it wrong to just choose a single version and demand consistency. I have no particular preference for including or excluding the space, I just want it uniformly chosen across the repo

Reply:
:+1: if it's reasonably easy to change (should just be a quick regex-replace) then I'd mildy prefer the `/// @{` syntax (with the space) for consistency with the other doxygen comments (e.g. `/// \cond`).